src/components/SykmeldingViews/SykmeldingView/Sections/SykmeldingViewSykmeldt/MeldingTilNav.tsx

```diff
@@ -1,33 +1,29 @@
 import { PeopleDialogOutline } from '@navikt/ds-icons'
 
 import { MeldingTilNav } from '../../../../../fetching/graphql.generated'
-import JaEntry from '../../Layout/JaEntry/JaEntry'
-import SykmeldingEntry from '../../Layout/SykmeldingEntry/SykmeldingEntry'
-import { SykmeldingSectionHeading } from '../../../../molecules/sykmelding/SykmeldingGroup'
+import { SykmeldingGroup } from '../../../../molecules/sykmelding/SykmeldingGroup'
+import { SykmeldingInfo, SykmeldingJaInfo } from '../../../../molecules/sykmelding/SykmeldingInfo'
 
 interface Props {
     meldingTilNav?: MeldingTilNav | null
 }
 
 function MeldingTilNav({ meldingTilNav }: Props): JSX.Element | null {
-    if (!meldingTilNav || (meldingTilNav.bistandUmiddelbart === false && !meldingTilNav.beskrivBistand)) {
+    if (meldingTilNav == null || (!meldingTilNav.bistandUmiddelbart && meldingTilNav.beskrivBistand == null)) {
```

Comment:
`!meldingTilNav` og `meldingTilNav == null`, går for det sammme, men meir leselig?

Reply:
`!thing` og `thing == null` er det samme dersom det er objekter eller arrays (`Thingy[] | null | undefined` eller `Thingy | null | undefined` f.eks). Men det er ikke det samme for primitiver som string og number. Da vil falsey også ta med tallet `0` og tom streng `""`.

Så stort sett så er jeg mer fan av å være eksplisitt på det, greit å sette `null` og `undefined` i samme bås, men å hive med tom streng og 0 kan være litt skummelt. :P